### DISCUSSION TITLE
绳网社区管理建议

### DISCUSSION THREAD

**Discussion by nostalgiatan** [share121/inter-knot#813] [其他]
1、发布什么内容，建议前面加括号前置【】，方便后期分类处理，这是之后可能会出现的菜单

首页（随机出现帖子）
咖啡馆（注水帖子）
电玩城（游戏分享）
茶亭（官方资讯）
治安局（反馈与建议）

2、反馈与建议，建议一律以议题的形式发送，方便管理者收集意见

**Comment by keerdun** [comment]
其实我还是挺喜欢这种乱七八糟的，但支持

**Comment by SDCOM-0415** [comment]
支持

**Comment by xiaochenyu24** [comment]
支持

**Comment by aaa00073** [comment]
我还以为咖啡馆难道指的是我发的咖啡馆恐怖小说，[笑哭]吓死我了还以为被注意到并被总结了。[哈哈]
